apps/core/src/lifecycle-hooks/dynamodb-local-setup.ts

```diff
@@ -30,6 +30,9 @@ export class DynamoDbLocalSetupService implements OnApplicationBootstrap {
   ) {}
 
   async onApplicationBootstrap() {
+    if (!this.dbConfig.launchLocal) {
+      return;
+    }
```

Comment:
Nit: Could you add a comment or abstract into a symbol?

Reply:
will do 👍